docs/cli/run_create_start_stop.md

```diff
@@ -97,6 +97,7 @@ ones common to both subcommands are described below:
                                      (/absolute/path/on/VM:path/on/host)
 - `--copy-timeout COPY_TIMEOUT`: The maximum time to wait for a copy-in-before or
                                  copy-out-after operation to complete
+- `--copy-using-image`:    Use the target image's session for copying instead of a new QEMU instance
```

Comment:
I think this should probably be named differently. Maybe `--direct-copy` with a description of  "Transfer files specified via `--copy-in-before` or `--copy-out-after` directly to the VM, instead of a background 'builder' VM"

Reply:
Done.   (I had trouble finding a name for this thing and that sounds much better!)